ooniprobe/View/RunTestViewController.h

```diff
@@ -0,0 +1,33 @@
+//
+//  RunTestTableViewController.h
+//  ooniprobe
+//
+//  Created by Lorenzo Primiterra on 06/08/17.
+//  Copyright © 2017 Simone Basso. All rights reserved.
+//
```

Comment:
Same comment about the copyright

Reply:
(Uhm, maybe we can change something in the Xcode config so to get it by default?)

Reply:
If you know this settings let me know !